Gaijinizer patch/Map293.txt

```diff
@@ -95,12 +90,10 @@ Beetle Harpy...#GT
 
 そして、その鋭い牙や爪で血肉を貪る………。
 > CONTEXT: Map293/events/1/pages/0/41/Dialogue/enemy4
-............ That devil is a pure carnivore
-It's a weak force to resist, hurt injured
-people and girls and children.
-
-And I slap blood with its sharp
-fangs and nails.......#GT
+...Those demons are pure carnivores,
+feeding on the weak, injured, women, and children.
+And then they gorge on their kills with
+sharp teeth and fangs...#Amaboye
 > END STRING
```

Comment:
Minor rewording:
```
...Those demons are pure carnivores.
They feed on the weak and injured as well as
women and children, gorging on their kills with
sharp fangs and claws...
```
(teeth and fangs are technically the same thing, and I got claws out of DeepL.